### DISCUSSION TITLE
Pagination is not align with Select Component

### DISCUSSION THREAD

**Discussion by mazipan** [mantinedev/mantine#2001] [Feedback]
Somehow the pagination size is not align with Select component.
If we pass the same `size` props.
It's kind a weird when showing these two component inline.

<img width="628" alt="Screen Shot 2022-08-04 at 08 49 34" src="https://user-images.githubusercontent.com/7221389/182745760-82a8ccbd-3a9b-41ea-afaf-2038317e5e85.png">

**Comment by rtivital** [comment]
You can set pagination size as number, for example `<Pagination size={50} />` to make it the same height as Select component. Pagination uses ActionIcon component under the hood – its sizes do not match inputs.

**Reply by mazipan** [reply]
I use this workaround btw:

```ts
<Pagination
  sx={() => ({
    '> button': {
      height: '36px',
      minWidth: '36px',
    },
  })}
/>
```

**Reply by aalster** [reply]
Workaround not working for me with mantine v7.8.0. I'm frustrated right now spending so much time to line up basic components
